test/extensions/KHR/SPV_KHR_untyped_pointers/basic.ll

```diff
@@ -0,0 +1,35 @@
+; RUN: llvm-as %s -o %t.bc
```

Comment:
Are this and following few tests from a different patch?

Reply:
Oopsy, yes, will remove